content/en/monitors/create/types/ci_pipelines.md

```diff
@@ -0,0 +1,104 @@
+4. Group CI Pipeline events by multiple dimensions (optional):
+    All CI Pipeline events matching the query are aggregated into groups based on the value of up to four facets.
+5. Configure the alerting grouping strategy (optional):
+    * If the query has a `group by`, multi alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you could group a query by `@ci.pipeline.name` to receive a separate alert for each CI Pipeline name when the number of errors is high.
+
+{{< img src="monitors/monitor_types/ci_pipelines/define-the-search-query.png" alt="Define the search query" style="width:80%;" >}}
+
+#### Using formulas and functions
+
+You can create CI Pipeline monitors using formulas and functions. This can be used for example to create monitors on the **rate** of an event happening, such as the rate of a pipeline failing (error rate).
+
+The following example is of a pipeline error rate monitor using a formula that calculates the ratio of "number of failed pipeline events" (`ci.status=error`) over "number of total pipeline events" (no filter), grouped by `ci.pipeline.name` (to be alerted once per pipeline).
+
+{{< img src="monitors/monitor_types/ci_pipelines/define-the-search-query-fnf.png" alt="Define the search query" style="width:80%;" >}}
+
+<div class="alert alert-info"><strong>Note</strong>: Only up to 2 queries can be used to build the evaluation formula per monitor.</div>
+
+### Set alert conditions
+
+* Trigger when the metric is `above`, `above or equal to`, `below`, or `below or equal to`
+* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between `1 minute` and `2 days`.
+* Alert threshold `<NUMBER>`
+* Warning threshold `<NUMBER>`
+
+#### Advanced alert conditions
+
+For detailed instructions on the advanced alert options (evaluation delay, etc.), see the [Monitor configuration][3] page.
+
+### Notifications
+
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][4] page.
+
+#### Notifications behavior when there is no data
+
+A monitor that uses an event count, or a formula for its evaluation query will resolve after the specified evaluation period with no data, triggering a notification. For example, a monitor using a formula to alert on pipeline error rate with an evaluation window of 5 minutes will automatically resolve after 5 minutes without any data.
+
+As CI pipeline data is usually sparse and can have relatively long periods with no data, this might result in monitor recovery notifications that might not be desired.
```

Comment:
```suggestion
As CI pipeline data is usually sparse and can have relatively long periods with no data, this can result in monitor recovery notifications that might not be desired.
```